src/camera/Camera.ts

```diff
@@ -112,11 +102,69 @@ abstract class Camera {
     return this;
   }
 
+  public getPrevAnchor(anchor: AnchorPoint): AnchorPoint | null {
+    return this._anchors[anchor.index - 1] || null;
+  }
+
+  public getNextAnchor(anchor: AnchorPoint): AnchorPoint | null {
+    return this._anchors[anchor.index + 1] || null;
+  }
+
+  public findAnchorIncludePosition(position: number): AnchorPoint | null {
+    const anchors = this._anchors;
+
+    for (const anchor of anchors) {
+      if (anchor.panel.includePosition(position, true)) {
+        return anchor;
+      }
+    }
+
+    return null;
+  }
+
+  public findNearestAnchor(position: number): AnchorPoint | null {
+    const anchors = this._anchors;
+
+    if (anchors.length <= 0) return null;
```

Comment:
It would be nice to make a constant called length.

Reply:
Okay, I'll do that.